app/forms/provider_interface/offer_wizard.rb

```diff
@@ -43,11 +43,27 @@ def self.build_from_application_choice(state_store, application_choice, options
         standard_conditions: standard_conditions_from(application_choice.offer),
         further_condition_attrs: further_condition_attrs_from(application_choice.offer),
         ske_conditions: ske_conditions_from(application_choice.offer),
+        require_references: require_references_from(application_choice.offer),
+        references_description: references_description_from(application_choice.offer),
       }.merge(options)
 
       new(state_store, attrs)
     end
 
+    def self.require_references_from(offer)
+      return REQUIRE_REFERENCES_CHECKED_BY_DEFAULT if reference_condition(offer)&.required.present?
+
+      '0'
```

Comment:
Well, I need to investigate a way of setting true or false instead 1 and 0 for checkboxes on single checkboxes. If is not possible, needs to make this more prettier.